shared/src/commonMain/kotlin/vn/luongvo/kmm/survey/data/local/datasource/SurveyLocalDataSource.kt

```diff
@@ -0,0 +1,37 @@
+package vn.luongvo.kmm.survey.data.local.datasource
+
+import io.realm.kotlin.Realm
+import io.realm.kotlin.UpdatePolicy
+import io.realm.kotlin.ext.query
+import vn.luongvo.kmm.survey.data.local.model.SurveyRealmObject
+
+interface SurveyLocalDataSource {
+
```

Comment:
Please remove this break line ([our convention](https://nimblehq.co/compass/development/code-conventions/kotlin/#convention-5))

Reply:
@suho I think it's not a big deal. However, better to follow the convention. 🤷‍♂️

Reply:
@suho Fixed in https://github.com/luongvo/kmm-survey/pull/89/commits/8ab4b50acc7042828b85903d96785a791d292947